app/src/main/java/agency/five/codebase/android/movieapp/ui/component/CrewmanCard.kt

```diff
@@ -0,0 +1,76 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.ui.theme.MovieAppTheme
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+data class CrewmanCardCardViewState(
+    val name: String,
+    val role: String,
+)
+
+@Composable
+fun CrewmanCard(
+    crewmanCardCardViewState: CrewmanCardCardViewState,
+    modifier: Modifier = Modifier,
+) {
+    Surface(color = MaterialTheme.colors.background,
+        modifier = Modifier.padding(10.dp)) {
+        Card {
+            Column(
+                modifier = Modifier
+                    .wrapContentSize()
+                    .clip(RoundedCornerShape(4.dp)),
+            ) {
+                Text(text = "${crewmanCardCardViewState.name}",
+                    modifier = Modifier.padding(start = 5.dp,
+                        end = 5.dp,
+                        top = 3.dp,
+                        bottom = 3.dp),
+                    fontWeight = FontWeight.Bold
+                )
+
+                Text(
+                    text = "${crewmanCardCardViewState.role}",
+                    modifier = Modifier.padding(start = 5.dp,
+                        end = 5.dp,
+                        top = 3.dp,
+                        bottom = 5.dp),
+                )
+            }
+        }
+    }
+}
+
+@Preview
+@Composable
+private fun CrewmanCardPreview(){
+    MovieAppTheme {
+        Surface(color = MaterialTheme.colors.background,
+            modifier = Modifier.padding(10.dp)) {
+            Card {
+                Column(
+                    modifier = Modifier
+                        .wrapContentSize()
+                        .clip(RoundedCornerShape(4.dp)),
+                ) {
+                    CrewmanCard(crewmanCardCardViewState = CrewmanCardCardViewState(
```

Comment:
when you have multiple paramaters, every parameter should be in the new line (including first one) - please go through the all function calls and reformat this

Reply:
Done